site/app/templates/calendar/CalendarOptions.twig

```diff
@@ -7,17 +7,17 @@
         {% if course_names|length > 1 %}
         <option value="show all">All Courses</option>
         {% endif %}
-        {% for course in course_names %}
-        <option value="{{course}}">{{course}}</option>
+        {% for item in course_names %}
+        <option value="{{item.course}}">{{item.name}}</option>
```

Comment:
`item` isn't a very descriptive identifier.  The original `course` was much more descriptive.  Can you please choose a better identifier?